src/test/java/emissary/directory/RoutingAlgorithmTest.java

```diff
@@ -260,11 +260,11 @@ public void testErrorHandlingErrorPopsAllCurrentForms() {
         loadAllTestEntries();
         final DirectoryEntry eplace = new DirectoryEntry("ERROR.e1.IO.http://example.com:8001/E$5050");
         this.dir.addTestEntry(eplace);
-        this.payload.pushCurrentForm(emissary.core.Form.ERROR);
-        this.payload.pushCurrentForm(emissary.core.Form.ERROR);
+        this.payload.pushCurrentForm("OTHER1");
+        this.payload.pushCurrentForm("OTHER2");
         this.payload.pushCurrentForm(emissary.core.Form.ERROR);
         final DirectoryEntry result = this.agent.getNextKeyAccess(this.dir, this.payload);
-        assertEquals("Error in error handling place removes all forms", 0, this.payload.currentFormSize());
+        assertEquals("Error in error handling place removes all other forms", 1, this.payload.currentFormSize());
```

Comment:
What should happen here? Should all the forms be cleared out, or should it just be left with the Error form?